docs/Makefile

```diff
@@ -0,0 +1,3 @@
+ams-schema-spec.html: ams-schema-spec.bs
+	bikeshed spec $< $@
+	sed -i 's/ *$$//' $@
```

Comment:
This does almost the same as the `whitespace` task from pre-commit. It's meant to make `git add -p` less painful.

Reply:
👍🏼